config/aso/kustomization.yaml

```diff
@@ -6,80 +6,79 @@ resources:
 - crds.yaml
 - settings.yaml
 
-patches:
-  - path: patches/visualizer_label_in_bastionhosts.yaml
-  - path: patches/visualizer_label_in_extensions.yaml
-  - path: patches/visualizer_label_in_fleetmembers.yaml
-  - path: patches/visualizer_label_in_managedclusteragentpools.yaml
-  - path: patches/visualizer_label_in_managed_clusters.yaml
-  - path: patches/visualizer_label_in_natgateways.yaml
-  - path: patches/visualizer_label_in_privateendpoints.yaml
-  - path: patches/visualizer_label_in_resourcegroups.yaml
-  - path: patches/visualizer_label_in_subnets.yaml
-  - path: patches/visualizer_label_in_virtualnetworks.yaml
-  - patch: |- # default kustomization includes a namespace already
```

Comment:
If you let 'kustomize edit fix' run, it removed comments like these that we probably want to retain. There are only a few of them, but you'll have to inspect the diff. (See #4753 for comparison.)